src/Chapter1.hs

```diff
@@ -508,8 +508,12 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
-
+--closestToZero x y = min (abs (x - 0)) (abs (y - 0))
+closestToZero x y = if (abs(x-0) < abs(y-0))
```

Comment:
You can omit subtracting 0 as it gives no difference when working with `Int`s 🙂 

Also you can remove parenthesis in the `if` condition:
```suggestion
closestToZero x y = if abs x < abs y
```